environments/eu/public.yml

```diff
@@ -0,0 +1,223 @@
+  CELERY_BROKER_HEARTBEAT: 600
+  #  COUCH_CACHE_DOCS:
+  #  COUCH_CACHE_VIEWS:
+  CONNECTID_URL: "https://connectid.dimagi.com/o/userinfo"
+  COUCH_USERNAME: "{{ COUCH_USERNAME }}"
+  COUCH_PASSWORD: "{{ COUCH_PASSWORD }}"
+  DEPLOY_MACHINE_NAME: "{{ inventory_hostname }}"
+  DEVICE_LOGS_ENABLED: no
+  DEVICE_LIMIT_PER_USER: 50
+  EMAIL_SMTP_HOST: email-smtp.us-east-1.amazonaws.com
+  EMAIL_SMTP_PORT: 587
+  EMAIL_USE_TLS: yes
+  ENABLE_SOFT_ASSERT_EMAILS: True
+  ENFORCE_SSO_LOGIN: True
+  ELASTICSEARCH_MAJOR_VERSION: 5
+  ES_CASE_SEARCH_INDEX_NAME: "case_search_2022-10-14"
+  ES_MULTIPLEX_TO_VERSION: "6"
+  # Index Multiplexer Settings
+  ES_APPS_INDEX_MULTIPLEXED: False
+  ES_CASE_SEARCH_INDEX_MULTIPLEXED: False
+  ES_CASES_INDEX_MULTIPLEXED: False
+  ES_DOMAINS_INDEX_MULTIPLEXED: False
+  ES_FORMS_INDEX_MULTIPLEXED: False
+  ES_GROUPS_INDEX_MULTIPLEXED: False
+  ES_SMS_INDEX_MULTIPLEXED: False
+  ES_USERS_INDEX_MULTIPLEXED: False
+  # Index Swap Settings
+  ES_APPS_INDEX_SWAPPED: True
+  ES_CASE_SEARCH_INDEX_SWAPPED: True
+  ES_CASES_INDEX_SWAPPED: True
+  ES_DOMAINS_INDEX_SWAPPED: True
+  ES_FORMS_INDEX_SWAPPED: True
+  ES_GROUPS_INDEX_SWAPPED: True
+  ES_SMS_INDEX_SWAPPED: True
+  ES_USERS_INDEX_SWAPPED: True
+  # Index Settings End
+  IS_DIMAGI_ENVIRONMENT: True
+  EULA_COMPLIANCE: True
+  HQ_INSTANCE: "eu"
+  INACTIVITY_TIMEOUT: 20160
+  INVOICE_FROM_ADDRESS:
+    "name": "Dimagi, Inc."
+    "first_line": "585 Massachusetts Ave"
+    "city": "Cambridge"
+    "region": "MA"
+    "postal_code": "02139"
+    "country": "US"
+    "phone_number": "(617) 649-2214"
+    "email": "accounts@dimagi.com"
+    "website": "https://www.dimagi.com"
+  INVOICE_PREFIX: EU-
+  INVOICE_STARTING_NUMBER: 5000
+  J2ME_ADDRESS: "{{ J2ME_SITE_HOST }}"
+  LOCAL_CUSTOM_DB_ROUTING:
+    auditcare: auditcare
+    repeaters: repeaters
+  LOCAL_MIDDLEWARE:
+    - "django.middleware.security.SecurityMiddleware"
+  PILLOWTOP_MACHINE_ID: eucloud
+  RATE_LIMIT_SUBMISSIONS: yes
+  SES_CONFIGURATION_SET: eu-email-events
+  SNS_EMAIL_EVENT_SECRET: "{{ SNS_EMAIL_EVENT_SECRET }}"
+  REDIS_DB: "0"
+  REDIS_HOST: "redis.eu.commcare.local"
+  REDIS_PORT: "6379"
+  REMINDERS_QUEUE_ENABLED: True
+  REQUIRE_TWO_FACTOR_FOR_SUPERUSERS: True
+  SAVED_EXPORT_ACCESS_CUTOFF: 180
+  SMS_GATEWAY_URL: ""
+  SMS_QUEUE_ENABLED: True
+  SUMOLOGIC_URL: "{{ SUMOLOGIC_URL }}"
+  SYNC_CASE_FOR_MESSAGING_ON_SAVE: False
+  #  STATIC_ROOT:
+  WS4REDIS_CONNECTION_HOST: "redis.eu.commcare.local"
+  USER_REPORTING_METADATA_BATCH_ENABLED: True
+
+commcare_cloud_root_user: ubuntu
+commcare_cloud_pem: ~/.ssh/id_rsa
+
+firehose_endpoint: "firehose.eu-west-1.amazonaws.com"
+kinesis_endpoint: "kinesis.eu-west-1.amazonaws.com"
+kinesis_flows:
+  - file_pattern: "/opt/data/formplayer/log/request_response.*log"
+    delivery_stream: "formplayer-request-response-logs-partitioned-eu"
+
+enable_cloudwatch_logs: true
+
+efs_shared_mount_dir: "shared{{ '_' ~ deploy_env if deploy_env != 'production' else '' }}"
```

Comment:
This references `production`. Is that intentional?

Reply:
Yes. That is intentional. I think we have append env name in shared path for all environments other than production.